meson.build

```diff
@@ -296,10 +298,17 @@ if not headers_only
 			rtdl_deps += libgcc
 		endif
 
+		if get_option('buildtype') == 'debug'
+			sanitize_args = ['-fsanitize=undefined']
+		else
+			sanitize_args = []
+		endif
```

Comment:
Meson has a `-Db_sanitize=undefined` built-in flag for this purpose.

Reply:
This will build the tests with `-fsanitize=undefined` which causes them to break (since there's nothing linked in to provide the ubsan hooks). I'm not sure how to deal with that.

Reply:
We can either override it it for the target or see about including the hooks

Reply:
We could build the hooks as a static library and then link them into both?

Reply:
that would work, yeah, as a static library with install: false or such

Reply:
That probably doesn't work since the mlibc namespace has hidden linkage.

For now, we could just pass `override_options: ['b_sanitize=none']`.

Reply:
Yeah, but there's probably a linker flag for that? `--exclude-libs` should hide the symbols defined by the static library

Reply:
> That probably doesn't work since the mlibc namespace has hidden linkage.

yes, but static libs don't do visibility (they're just archives of objects), so I'm pretty sure it would (worth trying anyway, it shouldn't be very hard)

Reply:
Well, the issue is that the ubsan hooks use `mlibc::infoLogger` etc. which is not visible outside `libc.so`.

Reply:
Fair enough. I guess we can see about enabling sanitizers for the host libc tests (when that PR is merged), just as another sanity check.